Sprint-3/revise/implement/isPrime.js

```diff
@@ -0,0 +1,22 @@
+function isPrime(num){
```

Comment:
Why not design the function to return `true` or `false` to indicate if the given number is a prime?